src/draw/lv_draw.c

```diff
@@ -154,19 +155,32 @@ void lv_draw_finalize_task_creation(lv_layer_t * layer, lv_draw_task_t * t)
     LV_PROFILER_END;
 }
 
+void lv_draw_wait_for_finish()
+{
+#if LV_USE_OS
+    lv_draw_unit_t * u = _draw_info.unit_head;
+    while(u) {
+        if(u->wait_for_finish_cb)
+            u->wait_for_finish_cb(u);
+        u = u->next;
+    }
+#endif
+}
+
 void lv_draw_dispatch(void)
 {
     LV_PROFILER_BEGIN;
-    bool render_running = false;
+    bool task_dispatched = false;
```

Comment:
It will be also `true` if no dispatch happened now, but there is an already running rendering.

Reply:
Yes, the logic did not changed.
I rename it just because it can mean: unit running (busy), or task is unsupported by the unit, or no more tasks in the queue.

So the name `task_dispatched` looks more representative.